backend/src/apiserver/client/kfam.go

```diff
@@ -0,0 +1,85 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/kubeflow/pipelines/backend/src/common/util"
+	"github.com/pkg/errors"
+)
+
+type KFAMInterface interface {
+	IsAuthorized(userIdentity string, namespace string) (bool, error)
+}
+
+type KFAMClient struct {
+	kfamServiceUrl string
+}
+
+type User struct {
+	Kind string
+	Name string
+}
+
+type RoleRef struct {
+	ApiGroup string
+	Kind     string
+	Name     string
+}
+
+type Binding struct {
+	User              User
+	ReferredNamespace string
+	RoleRef           RoleRef
+}
+
+type Bindings struct {
+	Bindings []Binding
+}
+
+const (
+	HTTP_TIMEOUT_SECONDS = 10
+)
+
+func (c *KFAMClient) IsAuthorized(userIdentity string, namespace string) (bool, error) {
```

Comment:
Please add tests for this function

Reply:
This func is basically talk to the KFAM service. To add tests, there are two options: 
1) fake a local KFAM http service. Some amount of work, might not be worth it.
2) separate the logic into two functions and test the part where it finds the namespace from the binding response. This part of the logic is trivial, if you think is important, I can add the test for this latter part.
WDYT?

Reply:
Testing this piece of code end to end is critical and we should do it. 
https://golang.org/pkg/net/http/httptest/ might help you with the test.

Reply:
SG. I will add the tests now.

Reply:
Done.